.gitignore

```diff
@@ -1,30 +1,35 @@
-#file types
-_ReSharper.*
-*.resharper
-*.DS_Store
-*.tlog
-*.vsp
-*.psess
-*.v11.suo
-*.user
-*.suo
-#*.Framework.XML
-
-#specific files
-RockJobSchedulerService.Xml
-web.[Cc]onnectionStrings.config
-desktop.ini
-Rock.XML
-pageviews.csv
-RockLogster.csv
-Quartz.xml
-StyleCop.Cache
-
-#specific folders
-[Oo]bj/
-RockWeb/Cache/*
-RockWeb/App_Data/PackageStaging
-/scaffolding.config
-*.sln.GhostDoc.xml
-
-Packages/*
+#file types
+_ReSharper.*
+*.resharper
+*.DS_Store
+*.tlog
+*.vsp
+*.psess
+*.v11.suo
+*.user
+*.suo
+#*.Framework.XML
+
+#specific files
+RockJobSchedulerService.Xml
+web.[Cc]onnectionStrings.config
+desktop.ini
+Rock.XML
+pageviews.csv
+RockLogster.csv
+Quartz.xml
+StyleCop.Cache
+
+#specific folders
+[Oo]bj/
+RockWeb/Cache/*
+RockWeb/App_Data/PackageStaging
+RockWeb/Bin/*

```

Comment:
Bit concerned about excluding all of RockWeb/Bin/* (Looks like this and the javacript deps below were the only change).